nix/README.md

```diff
@@ -0,0 +1,38 @@
+## Overview
+
+These are a collection of packages we need, or packages where we 
+want to control the exact version(s) of.
+
+The packages are imported through the `nix-shell` automatically. If you
+run NixOS, read the following section.
+
+## Adding adding the overlay(s)
+```
+mkdir -p ~/.config/nixpkgs/overlays
+ln -s $(pwd)/nix/mayastor-overlay.nix ~/.config/nixpkgs/overlays/rust-overlay.nix
+```
+
+Now the package is integrated natively:
+
+```
+nix-env -qaP libiscsi
+nixos.libiscsi  libiscsi-1.19.0
+gila@gixos ~/MayaStor>
```

Comment:
nit: the first line is missing a prompt and the last line has it though there should not be any (probably).

Reply:
Thanks! Fixed 👍